src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -277,19 +277,40 @@ default boolean deathSendImage() {
     @ConfigItem(
         keyName = "deathNotifMessage",
         name = "Notification Message",
-        description = "The message to be sent through the webhook. Use %USERNAME% to insert your username",
+        description = "The message to be sent through the webhook. Use %USERNAME% to insert your username, %VALUELOST% to insert the GE value of the stuff you lost",
         position = 18,
         section = deathSection
     )
     default String deathNotifyMessage() {
         return "%USERNAME% has died...";
     }
 
+    @ConfigItem(
+        keyName = "deathNotifPvpEnabled",
+        name = "Distinguish PvP deaths",
+        description = "Should the plugin use a different message for dying in PvP?",
```

Comment:
```suggestion
        description = "Show deaths to other players separately from deaths to NPCs",
```
not 100% on this one - there's a stackexchange post for this exact phrase, `separately from`